incubator/neo4j/README.md

```diff
@@ -0,0 +1,77 @@
+# Neo4j
+
+[Neo4j](https://neo4j.com/) is a highly scalable native graph database that leverages data relationships as first-class entities, helping enterprises build intelligent applications to meet today’s evolving data challenges.
+
+## TL;DR;
+
+```bash
+$ git clone git@github.com:mneedham/charts.git
+$ cd charts
+```
+
+```bash
+$ helm install incubator/neo4j
+```
+
+## Introduction
+
+This chart bootstraps a [Neo4j](https://github.com/neo4j/docker-neo4j) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
```

Comment:
1.4+: Really? We are going to support the latest two versions in the future, which are 1.6 and 1.7 at the moment.